src/public/js/vendor/Inspectlet.ts

```diff
@@ -0,0 +1,32 @@
+export {};
```

Comment:
Filenames should be case insensitive. Rename this to `inspectlet.ts`. It would be appreciated if we can proactively look at existing coding proactive in the repo so such feedback can be avoided

Reply:
https://github.com/jalantechnologies/boilerplate-mern/pull/34#discussion_r935478866
Sorry here when Ankit Sir mentioned changing it from Inspect to Inspectlet, I pasted it as it is, next time I will try to maintain code consistency.